packages/commercetools/composables/src/getters/orderGetters.ts

```diff
@@ -1,26 +1,69 @@
 import { UserOrderGetters, AgnosticOrderStatus } from '@vue-storefront/interfaces';
-import { Order, OrderState } from './../types/GraphQL';
+import { Order, OrderState, LineItem, Money, Address } from './../types/GraphQL';
 
 export const getOrderDate = (order: Order): string => order?.createdAt || '';
 
 export const getOrderId = (order: Order): string => order?.id || '';
 
+export const getOrderNumber = (order: Order): string => order?.orderNumber || getOrderId(order);
+
 const orderStatusMap = {
   [OrderState.Open]: AgnosticOrderStatus.Open,
   [OrderState.Confirmed]: AgnosticOrderStatus.Confirmed,
   [OrderState.Complete]: AgnosticOrderStatus.Complete,
   [OrderState.Cancelled]: AgnosticOrderStatus.Cancelled
 };
 
+const getPrice = (money: Money): number | null => money && money.centAmount ? money.centAmount / 100 : null;
+
 export const getOrderStatus = (order: Order): AgnosticOrderStatus | '' => order?.orderState ? orderStatusMap[order.orderState] : '';
 
-export const getOrderPrice = (order: Order): number | null => order ? order.totalPrice.centAmount / 100 : null;
+export const getOrderItems = (order: Order): LineItem[] => order?.lineItems || [];
+
+export const getOrderPrice = (order: Order): number | null => getPrice(order?.totalPrice);
+
+export const getOrderNetValue = (order: Order): number | null => getPrice(order?.taxedPrice?.totalNet);
+
+export const getOrderGrossValue = (order: Order): number | null => getPrice(order?.taxedPrice?.totalGross);
+
+export const getOrderTaxValue = (order: Order): number | null => getPrice(order?.taxedPrice?.taxPortions[0]?.amount);
+
+export const getOrderTaxRate = (order: Order): number => (order?.taxedPrice?.taxPortions[0]?.rate || 0) * 100;
```

Comment:
What if items in the order have different tax rates? I think this helper should return an array.